test/crd_checks.go

```diff
@@ -206,3 +207,10 @@ func CheckServiceState(client *ServingClients, name string, inState func(s *v1al
 func GetConfigMap(client *pkgTest.KubeClient) k8styped.ConfigMapInterface {
 	return client.Kube.CoreV1().ConfigMaps("knative-serving")
 }
+
+// Returns a func that evaluates if a deployment has scaled to 0 pods
+func DeploymentScaledToZeroFunc() func(d *apiv1beta1.Deployment) (bool, error) {
```

Comment:
this should ideally be in https://github.com/knative/pkg/blob/master/test/kube_checks.go as it is specific to kubernetes object check. Maybe even the GetConfigMap() above. 

You can attempt to fix this, or I can do it in a later PR

Reply:
I created https://github.com/knative/pkg/pull/236 - I think we can go ahead and merge this then once the pkg pr merges we can make a PR here to switch to using that for both methods.

Reply:
sg.